src/lib/operators/jit_operator/operators/jit_expression.cpp

```diff
@@ -1,13 +1,70 @@
-#include "jit_compute.hpp"
+#include "jit_expression.hpp"
+
+#include <string>
 
 #include "../jit_constant_mappings.hpp"
 #include "../jit_operations.hpp"
+#include "../jit_types.hpp"
 
 namespace opossum {
 
+// Returns the enum value (e.g., DataType::Int, DataType::String) of a data type defined in the DATA_TYPE_INFO sequence
+#define JIT_GET_ENUM_VALUE(index, s) APPEND_ENUM_NAMESPACE(_, _, BOOST_PP_TUPLE_ELEM(3, 1, BOOST_PP_SEQ_ELEM(index, s)))
+
+// Returns the data type (e.g., int32_t, std::string) of a data type defined in the DATA_TYPE_INFO sequence
+#define JIT_GET_DATA_TYPE(index, s) BOOST_PP_TUPLE_ELEM(3, 0, BOOST_PP_SEQ_ELEM(index, s))
+
+// Call the compute function with the correct template parameter if compute is called without a template parameter and
+// store the result in the runtime tuple.
+#define JIT_COMPUTE_CASE(r, types)                                         \
```

Comment:
This is also defined in jit_operations.hpp. Please clean up the macro use.

Reply:
Renamed `JIT_COMPUTE_CASE` macro to `JIT_EXPRESSION_COMPUTE_CASE` in this file.